### DISCUSSION TITLE
Password access and token authentication

### DISCUSSION THREAD

**Discussion by ellisrj** [community/community#24080] [Apps, API and Webhooks]
<p>Hello, I don’t use my GitHib account often, but when I do use GitHub Desktop (from Windows 10), and I can push my changes to my repository successfully, but I get an email telling me:</p>
<p><em>You recently used a password to access the repository at xxxx/xxxxx with git using git/2.19.2.windows.1.</em></p>
<p>When I go to the help page regarding this, I read:</p>
<p><em><strong>Workflows affected</strong></em></p>
<p>***    Command line Git access***<br>
***    Desktop applications using Git (GitHub Desktop is unaffected)***<br>
***    Any apps/services that access Git repositories on <a href="http://GitHub.com" rel="noopener nofollow ugc">GitHub.com</a> directly using your password***</p>
<p>So, is GitHub Desktop really unaffected? If so, why am I getting this message?</p>
<p>Thanks for your help, its probably obvious to those of you that operate in this realm frequently.</p>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/ellisrj">@ellisrj</a> thanks for the question! The reason GitHub Desktop was unaffected is because the application has always generated an OAuth token for authentication. We were never storing a basic username/password to use for authentication. This means that you wouldn’t have been logged out of GitHub Desktop when this change happened since it had a valid OAuth token associated with your GitHub account. If you have any specific questions about this let me know!</p>

**Comment by masadi** [comment]
![image](https://user-images.githubusercontent.com/5530849/206131043-1928baf4-6a35-49ad-890b-9ce3440b88e7.png)
How about this sir?
